### PR TITLE
tried mix  blend mode

### PR DESCRIPTION
**Pull request by sobiiya**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Face Effect</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <img src="face.jpg" alt="">
+        <img src="text.jpg" alt="">
```
Comment (faran4engg) [inline index.html:15]: Always add alt

File: style.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container {
+    position: relative;
+    background: #b81641;
+    width: 100%;
+    height: 100vh;
+    isolation: isolate;
```
Comment (faran4engg) [inline style.css:12]: What does this `isolation` property do ?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    object-fit: cover;
+}
+
+.container img:nth-child(2) {
+    mix-blend-mode: multiply;
```
Comment (faran4engg) [inline style.css:25]: What are the other possible options for `mix-blend-mode` ?

File: index.html (hunk 1 above)
Comment (sobiiya) [reply index.html:15]: okay

File: style.css (hunk 2 above)
Comment (sobiiya) [reply style.css:12]: if you don't want to use background property,isolation property used.
actually in this project isolation propery is not used I was tried all property and option of mix blend mode and forget to remove isolation property

File: style.css (hunk 3 above)
Comment (sobiiya) [reply style.css:25]: mix-blend-mode: normal,
mix-blend-mode: hard-light,
mix-blend-mode: difference these are some options

File: style.css (hunk 2 above)
Comment (faran4engg) [reply style.css:12]: then lets remove it  and keep it clean

**Review by faran4engg — APPROVED** [review]
minor changes, once fixed, you can merge !

File: style.css (hunk 2 above)
Comment (sobiiya) [reply style.css:12]: okay

**Pull request merged**